go.mod

```diff
@@ -9,23 +9,23 @@ require (
 	github.com/google/gofuzz v1.1.0
 	github.com/google/uuid v1.1.2
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
-	github.com/openshift/api v0.0.0-20210416104836-15c7ae797f3f
+	github.com/openshift/api v0.0.0-20211108165917-be1be0e89115
 	github.com/openshift/apiserver-library-go v0.0.0-20210415093535-9176fb31e5dc
-	github.com/openshift/build-machinery-go v0.0.0-20210423112049-9415d7ebd33e
+	github.com/openshift/build-machinery-go v0.0.0-20210712174854-1bb7fd1518d3
 	github.com/openshift/client-go v0.0.0-20210409155308-a8e62c60e930
 	github.com/openshift/library-go v0.0.0-20210414082648-6e767630a0dc
 	github.com/pkg/profile v1.4.0 // indirect
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
-	k8s.io/api v0.21.0
+	k8s.io/api v0.22.1
 	k8s.io/apiextensions-apiserver v0.21.0
```

Comment:
shouldn't all deps be on the same level `0.22.1` ?

Reply:
fixed